### ISSUE TITLE
[Package Request]: custom teamviewer 11

### ISSUE THREAD

**Issue by DevifyPro** [microsoft/winget-pkgs#156742]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://teamviewer.com
Publisher: Teamviewer AG
Package Name: Teamviewer 11 Host
Description: pcgen teamviewer installer
Package Version: 11
Installer URL: https://dl.teamviewer.com/download/version_11x/TeamViewer_Host_Setup-idcd4g3mu2.exe

**Comment by Utesgui** [comment]
winget isn't supposed to have customized/personalized versions of well known software i guess

**Comment by Dragon1573** [comment]
There's already `TeamViewer.TeamViewer.Hosts` provided in the repository. Could you please explain how you get this installer from the Download Page URL?

**Comment by DevifyPro** [comment]
Would it be a better solution to have it hosted in a personalized
repository? I just need it because i need to install different software
with winget on some new pc’s and i need that custom teamviewer.

**Comment by DevifyPro** [comment]
> There's already `TeamViewer.TeamViewer.Hosts` provided in the repository. Could you please explain how you get this installer from the Download Page URL?

Actually we can get the download link from pur account, since that installer is custom for our company. But apparently cant have custom installers on the public repository, not sure if i can have my source where to add the software

**Comment by Dragon1573** [comment]
> Would it be a better solution to have it hosted in a personalized repository?

YES! There're 2 recommended personal _WinGet_ source API provider for private packages. See [the doc](https://github.com/microsoft/winget-pkgs/blob/master/doc%2Fprivate%2FREADME.md).
